server/views/referrals/show/risksAndNeeds/partials/layout.njk

```diff
@@ -0,0 +1,42 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "moj/components/side-navigation/macro.njk" import mojSideNavigation %}
+
+{% extends "../../../../partials/layout.njk" %}
+
+{% block personBanner %}
+  {% include "../../../_personBanner.njk" %}
+{% endblock personBanner %}
+
+{% block backLink %}
+  {{ govukBackLink({
+    text: "Back to referrals",
+    href: "#"
```

Comment:
Should this go to one of the case lists? Maybe determining the destination is a little complicated for now though

Reply:
Yeah it should, I've just raised card 1269 on the board to keep track of this.